chalice/.chalice/config.json

```diff
@@ -56,6 +56,23 @@
         "DYNAMO_OUTPUT_TABLE_NAME": "dcp-matrix-service-output-table-predev",
         "S3_RESULTS_BUCKET": "dcp-matrix-service-results-predev"
       }
+    },
+    "prod": {
+      "api_gateway_stage": "prod",
+      "environment_variables": {
+        "DEPLOYMENT_STAGE": "prod",
+        "DSS_STAGE": "prod",
+        "MATRIX_VERSION": "52940c5",
```

Comment:
It'd be nice if we only deployed tags to prod

Reply:
For prod deployments, tagging with SEMVER version e.g. `v0.0.1`